server/src/main/java/org/opensearch/gateway/remote/ClusterStateDiffManifest.java

```diff
@@ -159,8 +160,9 @@ public ClusterStateDiffManifest(
         this.indicesDeleted = Collections.unmodifiableList(indicesDeleted);
         this.clusterBlocksUpdated = clusterBlocksUpdated;
         this.discoveryNodesUpdated = discoveryNodesUpdated;
-        this.indicesRoutingUpdated = Collections.unmodifiableList(indicesRoutingUpdated);
-        this.indicesRoutingDeleted = Collections.unmodifiableList(indicesRoutingDeleted);
+        this.indicesRoutingUpdated = indicesRoutingUpdated;
+        this.indicesRoutingDeleted = indicesRoutingDeleted;
```

Comment:
Why is this changed?

Reply:
It was left out in re-base. Addressed it.